persistence/selecting.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Selecting a persister
+summary: A guide for selecting an NServicebus persister.
+reviewed: 2019-02-27
+---
+
+NServiceBus needs to store data for various purposes. This document is written to help select the right persister for NServiceBus from the [list of available persisters](/persistence/#available-persisters).
+
+## Why does NServiceBus need storage?
+
+NServiceBus needs a persister to store the state of sagas, enable the outbox and, sometimes, to make a transport feature complete.
+
+- **Sagas**
+  A powerful way to orchestrate long-running business processes is provided by [sagas](/nservicebus/sagas/). They contain state that needs to be persisted.
+- **Outbox**
+  The [outbox](nservicebus/outbox/) needs to store message details to provide deduplication. See the [persisters documentation](http://nservicebus/outbox/#persistence) for more details.
```

Comment:
```suggestion
  The [outbox](nservicebus/outbox/) stores message details in order to prevent processing duplicated messages. See the [persistence documentation](http://nservicebus/outbox/#persistence) for more details.
```